test/compilable/interpret3.d

```diff
@@ -897,13 +897,6 @@ int staticdynamic()
 }
 static assert(staticdynamic() == 0);
 
-int[] crashing()
-{
-    int[12] cra;
-    return (cra[2 .. $] = 3);
-}
-static assert(crashing()[9] == 3);
```

Comment:
I removed it because it is a seriously broken piece of code, @safe or not. It is returning a pointer to its stack frame, and then dereferencing it. Allowing this kind of code in the name of backwards compatibility is a disservice to our users.